device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,285 @@
+{
+    "chassis": {
+        "name": "MSN2700",
+        "components": [],
+        "fans": [
+            {
+                "name": "fan1"
+            },
+            {
+                "name": "fan2"
+            },
+            {
+                "name": "fan3"
+            },
+            {
+                "name": "fan4"
+            },
+            {
+                "name": "fan5"
+            },
+            {
+                "name": "fan6"
+            },
+            {
+                "name": "fan7"
+            },
+            {
+                "name": "fan8"
+            }
+        ],
```

Comment:
Since these fans are not directly on the chassis, but are rather contained within fan drawers, I feel like the API should be changed to return 0 fans on the chassis. However, this is what is currently returned from the API.

Reply:
yes, for this specific SKU the fan is contained in drawers, not directly on chassis

Reply:
fixed, removed all fans under chassis